mesh/core/src/main/protobuf/resolver.proto

```diff
@@ -39,13 +39,7 @@ message Endpoint {
   int32 port = 3;
 
   /** Endpoint-specific diagonstic metadata */
-  message Meta {
-    /**
-     * In scheduled environments, a diagnostic name for the parent node.
-     */
-    string nodeName = 1;
```

Comment:
We should use field number 5 to preserve backwards compatibility.

Reply:
I guess we should also till be populating the field 4 as well.